src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DOUBLE_SIZE = 2;
+    private Node<K, V>[] buckets;
+    private boolean nullkey = false;
+    private V nullvalue;
+    private int size;
+
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        buckets = new Node[DEFAULT];
+    }
 
     @Override
     public void put(K key, V value) {
```

Comment:
resize first

Reply:
not fixed

Reply:
resize need to be before constructor myhashmap or before method put?